### PR TITLE
Feature: add RedShift

### PR DESCRIPTION
**Pull request by cyyeh**
## Description

Add Redshift as a data source

## Issue ticket number

None

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,7 +65,8 @@
     "toad-scheduler": "^2.2.0",
     "tslib": "^2.3.0",
     "tslog": "^3.3.3",
-    "uuid": "^9.0.0"
+    "uuid": "^9.0.0",
+    "@aws-sdk/client-redshift-data": "^3.405.0"
```
Comment (kokokuo) [inline package.json:69]: I may suggest you put the `@aws-sdk/client-redshift-data` package in the `extension-driver-redshift` package, then the `@aws-sdk/client-redshift-data` package will only installed when user add the `extension-driver-redshift`

You could refer to the `package.json`  of `extension-driver-clubhouse` and `extension-driver-snowflake` packages.

File: packages/extension-driver-redshift/package-lock.json
Comment (kokokuo) [inline packages/extension-driver-redshift/package-lock.json:1]: How about using the yarn to create the `yarn.lock` to follow our consistency like our packages?

File: packages/extension-driver-redshift/project.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      }
+    },
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["packages/extension-driver-redshift/**/*.ts"]
+      }
+    },
+    "test": {
```
Comment (kokokuo) [inline packages/extension-driver-redshift/project.json:22]: Seems missing the `publish` command. Btw, after putting the `@aws-sdk/client-redshift-data` package in the `extension-driver-redshift` vulcan package, still need the `install-dependencies` command.

Refer: https://github.com/cyyeh/vulcan-sql/blob/feature/add-redshift/packages/extension-driver-clickhouse/project.json

**Review by kokokuo — COMMENTED** [review]
Besides minor suggestions, others LGTM 👍  Great works!

**Comment by cyyeh** [comment]
Close the PR since it's from forked PR.

**Pull request closed without merging**